sdk/storage/README.md

```diff
@@ -0,0 +1,117 @@
+- **Source Code Contributions:** Please follow the [contribution guidelines for Azure open source](https://azure.github.io/azure-sdk/cpp_introduction.html) for instructions about contributing to the source project.
+
+For general suggestions about Azure, use our [Azure feedback forum](http://feedback.azure.com/forums/34192--general-feedback).
+
+## Download & Install
+
+### Install Dependencies
+
+#### Windows
+
+On Windows, dependencies are managed by [Vcpkg](https://github.com/microsoft/vcpkg). You can reference the [Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) to quickly set yourself up.
+After Vcpkg is initialized and bootstrapped, you can install the dependencies:
+```BatchFile
+vcpkg.exe install libxml2:x64-windows curl:x64-windows
+```
+
+#### Unix Platforms
+
+You can use the package manager on different Unix platforms to install the dependencies. The dependencies to installs are:
+
+CMake 3.15.0 and greater.
+Libxml2.
+OpenSSL.
+Libcurl.
+
+### Build from Source
+
+First, download the repository to your local folder:
+```BatchFile
+git clone https://github.com/Azure/azure-sdk-for-cpp.git
+```
+
+#### Windows
+
+##### Use CMake to generate the solution file
+```BatchFile
+cmake . -A x64 -DCMAKE_TOOLCHAIN_FILE=<YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake -DBUILD_CURL_TRANSPORT=ON -DBUILD_TESTING=ON -DBUILD_STORAGE_SAMPLES=ON -DCMAKE_CXX_FLAGS="/MP /EHsc"
+```
+
+Then you can open the generated `az.sln` to open the solution and build the library.
+The built library will be in `<ProjectDir>\<Setup>\` respectively for Azure Core and Azure Storage. e.g. `azure_core.lib` will be in `sdk\core\azure-core\Debug` for debug setup.
+
+##### Use Visual Studio's Open by folder feature
+Open the root folder of the library with Visual Studio's Open folder feature.
+
+If Vcpkg is not globally integrated, then you need to open CMakeSettings.json and change the `Make toolchain file to be <YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake` and save.
+Then you can build Azure Storage libraries by selecting the target in Visual Studio, or simply build all.
+The libraries will be in `<ProjectRoot>\out\build\<Setup>\sdk\<LibraryName>` respectively.
+
+#### Unix Platforms
+
+You can run the following command in the downloaded code's root folder to build the code.
+
+```BatchFile
+cmake . -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DBUILD_STORAGE_SAMPLES=ON
+make
+```
+Then you can consume the built library with the header files.
+make/ninja install is work in progress.
+
+
+### Via NuGet
+WIP
+TODO when ready.
+
+### Via Vcpkg
+WIP
+TODO when ready.
+
+## Dependencies
+
+### Azure Core SDK
+[Reference documentation.](https://github.com/Azure/azure-sdk-for-cpp/blob/master/README.md)
+
+### nlohmann/json
+[Official Web Page](https://github.com/nlohmann/json)
+
+### Libxml2
+[Official Web Page](http://xmlsoft.org/)
```

Comment:
[Azure Core SDK](https://github.com/Azure/azure-sdk-for-cpp/blob/master/README.md)

[nlohmann/json](https://github.com/nlohmann/json)

[Libxml2](http://xmlsoft.org/)